eng/service.proj

```diff
@@ -11,6 +11,8 @@
     <ExcludeMgmtLib Include="..\sdk\*\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj;" />
     <TestProjects Include="..\sdk\$(ServiceDirectory)\**\tests\**\*.csproj" />
     <SamplesProjects Include="..\sdk\$(ServiceDirectory)\**\samples\**\*.csproj" />
+    <!-- Exclude test projects that don't support net461 -->
+    <SamplesProjects Condition="'$(TargetFramework)' == 'net461'" Remove="..\sdk\core\Microsoft.Extensions.Azure\samples\Microsoft.Extensions.Azure.Samples.csproj" />
```

Comment:
I generally only like this approach when it is temporary and has an issue filed to fix. If this is never intended to work perhaps we should come up with an alternative strategy. Is there a reason this project cannot support net461?

Reply:
Yes, it's aspnet core app, ASP.NET Core doesn't support net461